packages/api/src/CallFetch.tsx

```diff
@@ -0,0 +1,34 @@
+export interface IHeaders {
```

Comment:
I think the filename should start with a lowercase, and its extension be just `ts` since there is no JSX inside. So `callFetch.ts`

Reply:
Maybe since we are here, also rename `IHeaders` to `IFetchHeaders`?